src/org/thoughtcrime/securesms/translation/TranslationTask.java

```diff
@@ -0,0 +1,64 @@
+package org.thoughtcrime.securesms.translation;
+
+import android.os.AsyncTask;
+import android.util.Log;
+import android.view.View;
+
+import org.thoughtcrime.securesms.database.model.MessageRecord;
+import org.thoughtcrime.securesms.util.JsonUtils;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class TranslationTask extends AsyncTask<String, Void, TranslationResult> {
+    private String API_KEY = "trnsl.1.1.20180413T005640Z.e9d9285313a77d68.f90bbb8e14d7b2bd9257f87c43139b70f49023e1";
```

Comment:
Going to deactivate this key and activate a new one. (And remove it in a future commit)